lib/takeaway.rb

```diff
@@ -0,0 +1,20 @@
+require_relative './menu.rb'
+require_relative './order.rb'
+
+class Takeaway
+  attr_reader :menu, :order
+
+  def initialize(menu)
+    @menu = Menu.new(menu)
+  end
+
+  def show_menu
+    @menu.show_menu
+  end
+
+  def place_order(dishes)
+    food.each do |dish, quantity|
```

Comment:
food.each just needs to be set to dishes, but this is already something you are aware of as mentioned on Zoom :)
```suggestion
    dishes.each do |dish, quantity|
```